### ISSUE TITLE
Release 4.0.1—supply chain attack?

### ISSUE THREAD

**Issue by sdavids** [azeemba/eslint-plugin-json#94]
https://www.npmjs.com/package/eslint-plugin-json?activeTab=versions

has a version `4.0.1`.

There have been no commits on this repo for 3 months ≈ release `4.0.0`.

This is how supply chain attacks happen …

17k downloads already …

---

Please provide a changelog:

https://keepachangelog.com/en/1.1.0/

and/or use GitHub releases:

https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

**Comment by sdavids** [comment]
#42

**Comment by azeemba** [comment]
#92 provides context. The README change was never published to npm resulting in stale documentation on npm. So the latest patch release `4.0.1` indeed matches the current state of the repo.

Though I should have pushed the version bump as well. Thanks for flagging that!

**Comment by azeemba** [comment]
I have pushed the version bump so will close the issue.

Just for extra validation on my claim, anyone can run `npm diff` to see what has changed between versions:

```sh
$ npm diff --diff=eslint-plugin-json@4.0.0 --diff=eslint-plugin-json@4.0.1 --diff-name-only
package.json
README.md
```

README contains the documentation changes and package.json has the version bump.

**Comment by sdavids** [comment]
So this is closed—duplicate of #42?

**Comment by sdavids** [comment]
Having a CHANGELOG (https://github.com/azeemba/eslint-plugin-json/issues/42) would be quite useful …
